tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/observability/tracing/MainFunctionTestCase.java

```diff
@@ -50,7 +50,6 @@ public void testMainMethod() throws Exception {
         final String span6Position = FILE_NAME + ":38:16";
         final String entryPointFunctionModule = "intg_tests/tracing_tests:0.0.1";
         final String entryPointFunctionPosition = "01_main_function.bal:19:1";
-        // TODO : Need to fix the position offset 55:43 (col 43 is not found in the source-code)
```

Comment:
did we find why the wrong file numbers are there ? is it not something wrong with the line-number table as we though?

Reply:
i couldnt reproduce the same invalid column lines once the instrumentation was updated. I need to look into it more.